### DISCUSSION TITLE
About TKOpenGL, When I create a View, the following error occurs

### DISCUSSION THREAD

**Discussion by WY-Huang** [Open-Cascade-SAS/OCCT#116] [Q&A]
TKOpenGl | Type: Error | ID: 0 | Severity: High | Message:
  Error: glBufferData (GL_ARRAY_BUFFER,4056,0x284B9880000) Id: 2 failed with GL_INVALID_OPERATION
TKOpenGl | Type: Performance | ID: 0 | Severity: Low | Message:
  VBO creation for Primitive Array has failed for 169 vertices. Out of memory?
TKOpenGl | Type: Error | ID: 0 | Severity: High | Message:
  Error: 2D texture 228x604 IF: GL_SRGB8_ALPHA8 PF: GL_RGBA DT: GL_UNSIGNED_BYTE can not be created with error GL_INVALID_OPERATION [fbo0_main:color]
TKOpenGl | Type: Error | ID: 0 | Severity: High | Message:
  Error! Main FBO 228x604@0 initialization has failed

**Comment by dpasukhi** [comment]
I will move the issue to discussion.
@WY-Huang Don't you mind?

Additionally, could you share the OS and available hardware? OCCT version and enabled 3rd parties.

**Comment by WY-Huang** [comment]
Not at all!
Thank you! The issue has been resolved.

**Comment by dpasukhi** [comment]
@WY-Huang great! Could you share your fix or just way to investigate the cause?
